archesproject/arches
# adds primeicons #11356

### Types of changes


### Description of Change

This adds support for primeicons


To test:

1. Checkout this branch
2. In a project, either point to this branch in `package.json` for the arches dependency, or manually the the `primeicons` dependency.
3. Set up a Vue application, or use an already-existing one
4. Render an icon with `<i class="pi pi-${something}></i>`
5. Notice the icon renders
